src/controllers/maps/Maps.ts

```diff
@@ -429,7 +429,7 @@ export default class Maps {
      */
 
     private static async ReplaceCode(timestamp: any): Promise<any> {
-        const regex_code = new RegExp(/<code>((\d+\|\d)(,(\d+\|\d))*)<\/code>/g);
+        const regex_code = /<code>((\d+\|\d(,\d+\|\d)*)|(\d+))<\/code>/g
```

Comment:
This doesn't seem to be removing the code tags from the link.

![image](https://user-images.githubusercontent.com/20389565/111076637-846a9a00-84c3-11eb-831a-cfcb10153377.png)

Reply:
It turns out the split function was necessary